.drone.star

```diff
@@ -53,22 +44,7 @@ config = {
                 "firefox",
             ],
             "phpVersions": [
-                "7.2",
                 "7.3",
-            ],
-        },
-        "webUI-74": {
-            "suites": {
-                "webUITwoFactorTOTP": "webUITwoFactTOTP",
-            },
-            "browsers": [
-                "chrome",
-                "firefox",
-            ],
-            "servers": [
-                "daily-master-qa",
-            ],
-            "phpVersions": [
```

Comment:
Do we really want ot remove these?

Reply:
It was running PHP 7.2 and 7.3 with the "default" servers (which is "latest" and "daily-master-qa") and also running PHP 7.4 with only "daily-master-qa".

I removed PHP 7.2. And put PHP 7,4 into the section that runs with the "default" serves. So actually more PHP 7,4 pipelines will run.